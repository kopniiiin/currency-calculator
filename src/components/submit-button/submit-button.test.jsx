import React from "react";
import {shallow} from "enzyme";

import SubmitButton from "./submit-button.jsx";

describe(`snapshot test: SubmitButton`, () => {
  it(`should render correctly`, () => expect(
    shallow(<SubmitButton label={`button`}/>)
  ).toMatchSnapshot());
});
