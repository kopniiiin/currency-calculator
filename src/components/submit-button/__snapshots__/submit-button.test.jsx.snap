// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test: SubmitButton should render correctly 1`] = `
<button
  aria-label="button"
  className=" submit-button"
  type="submit"
/>
`;
